dyscover/tests.py

```diff
@@ -1,3 +1,6 @@
 from django.test import TestCase
 
-# Create your tests here.
+class DyscoverViewsTestCase(TestCase):
+    def test_index(self):
+        resp = self.client.get('/api/search/?q=delhi')
+        self.assertEqual(resp.status_code, 200)
```

Comment:
yeah..checked those too individually... i will do them at once in the next commit